### PR TITLE
Fix some logging issues noticed in cromwell-dev

### PR DESCRIPTION
**Pull request by scottfrazer**
I just needed to get this committed somewhere because some of these issues were pain points in debugging the cromwell-dev instance

### CODE REVIEW

File: src/main/scala/cromwell/engine/backend/jes/Pipeline.scala [hunk 1]
```diff
@@ -8,11 +7,10 @@ import com.google.api.services.genomics.Genomics
 import com.google.api.services.genomics.model.CreatePipelineRequest
 import JesBackend._
 
-
-
 object Pipeline extends LazyLogging {
   def apply(command: String, workflow: WorkflowDescriptor, call: Call, jesParameters: Seq[JesParameter], projectId: String, jesConnection: JesInterface): Pipeline = {
-    logger.debug(s"Command line is $command")
+    val tag = s"JES Pipeline [UUID(${workflow.shortId}):${call.name}]"
+    logger.debug(s"$tag Command line is: $command")
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: Preexisting issue, but could we develop a standard name for our loggers?  I think `log` is the most common, but these changes perturbed code around `logger` and `Log` as well.

File: src/main/scala/cromwell/Main.scala [hunk 2]
```diff
@@ -25,10 +25,8 @@ object Main extends App {
   lazy val Log = LoggerFactory.getLogger("main")
 
   Option(Properties.getProperty(LoggerProperty)) match {
-    case None => args.headOption.map {_.capitalize}.find {_ == "SERVER"} match {
-      case Some(x) => Properties.setProperty(LoggerProperty, "SERVER")
```
Comment (mcovarr) [inline src/main/scala/cromwell/Main.scala]: ouch

**Comment by mcovarr** [comment]
:+1: from me, kicking it to @cjllanwarne

File: src/main/scala/cromwell/engine/backend/jes/Pipeline.scala (hunk 1 above)
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: I like `log` personally, `Log` second.  I suspect that it probably should be `Log` according to Scala conventions?  What do you think?

Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: I am also on Team `log`.

Comment (kshakir) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: Votes aside, isn't `logger` "hard coded" from TypeSafe's [`LazyLogging`](https://github.com/typesafehub/scala-logging/blob/v3.1.0/src/main/scala/com/typesafe/scalalogging/Logging.scala#L27)?

Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: Pretty solid argument for `logger`.  :smile:

Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: @kshakir good point

Comment (geoffjentry) [inline src/main/scala/cromwell/engine/backend/jes/Pipeline.scala]: I don't have intellij in front of me so take this with a grain of salt but I think that some of the problem is that the logging bits are coming from multiple places

**Comment by cjllanwarne** [comment]
Looks good to me

**Pull request merged**